src/charselect.cpp

```diff
@@ -174,6 +174,11 @@ void Courtroom::char_clicked(int n_char)
     ao_app->send_server_packet(
         new AOPacket("CC#" + QString::number(ao_app->s_pv) + "#" +
                      QString::number(n_char) + "#" + get_hdid() + "#%"));
+    if (n_char == -1) {
+      update_character(n_char);
+      enter_courtroom();
+      set_courtroom_size();
+    }
```

Comment:
Bruh you did not need to do that

```cpp
  if (n_char != m_cid || n_char == -1) {
    ao_app->send_server_packet(
        new AOPacket("PW#" + ui_char_password->text() + "#%"));
    ao_app->send_server_packet(
        new AOPacket("CC#" + QString::number(ao_app->s_pv) + "#" +
                     QString::number(n_char) + "#" + get_hdid() + "#%"));
  }

  if (n_char == m_cid || n_char == -1) {
    update_character(n_char);
    enter_courtroom();
    set_courtroom_size();
  }
```

Reply:
you're right im sorry i have terminal dumb bitch disease its incurable